packages/story-editor/src/components/mediaRecording/durationIndicator.js

```diff
@@ -53,7 +53,7 @@ const Wrapper = styled(Text).attrs({
 function DurationIndicator() {
   const { duration, isRecording } = useMediaRecording(({ state }) => ({
     duration: state.duration,
-    isRecording: state.status === 'recording',
+    isRecording: ['recording', 'paused'].includes(state.status),
```

Comment:
Nit: Would be nice to have the statuses in a constant since these are used in multiple places.

Reply:
Yeah true. Using TypeScript would be nice here